app/controllers/orders_controller.rb

```diff
@@ -16,6 +16,7 @@ def create
     order = current_user.orders.create(order_params)
     if order.save
       cart.items.each do |item,quantity|
+        new_quantity = (item.inventory - quantity)
```

Comment:
Where is this getting saved? It looks like this will only store the number as a variable and not actually remove the quantity from the inventory.